e2e/apprunner/copilot/front-end/manifest.yml

```diff
@@ -28,4 +28,9 @@ network:
 cpu: 1024
 # Amount of memory in MiB used by the task.
 memory: 2048
-# Number of tasks that should be running in your service.
\ No newline at end of file
+# Number of tasks that should be running in your service.
+
+secrets:                      # Pass secrets from AWS Systems Manager (SSM) Parameter Store and Secrets Manager.
+  my-ssm-secret: "arn:aws:ssm:us-east-1:067305307211:parameter/test-ssm-secret"
```

Comment:
this secret only exist in 067305307211 account which means the e2e test only works for that account. What I would expect from the e2e test is:

1. Create a secret using AWS CLI like https://github.com/aws/copilot-cli/blob/316aa07072b0b47baef5ce5efbaadd72710fcc21/e2e/sidecars/sidecars_suite_test.go#L47 in `BeforeSuite`
2. Use the secret in any of `back-end` or `front-end`
3. In `apprunner_test.go` do `svc show` and see if the secret is correctly  shown in the response https://github.com/aws/copilot-cli/blob/316aa07072b0b47baef5ce5efbaadd72710fcc21/e2e/sidecars/sidecars_test.go#L237
4. In `AfterSuite` remember to delete the secret.
5.